### ISSUE TITLE
Docs: Checkbox & Radio examples missing aria-hidden on SVG

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#1730]
SVGs are missing the aria-hidden="true" attribute.

**Comment by ianmcburnie** [comment]
Fixed in 13.7.0
